test/functional/feature_config_args.py

```diff
@@ -153,6 +153,12 @@ def test_invalid_command_line_options(self):
             expected_msg='Error: Error parsing command line arguments: Can not set -proxy with no value. Please specify value with -proxy=value.',
             extra_args=['-proxy'],
         )
+        # Provide a non-boolean-convertible arg to a negated option
+        if self.is_wallet_compiled():
+            self.nodes[0].assert_start_raises_init_error(
+                expected_msg="Error: Invalid value detected for '-wallet' or '-nowallet'. '-wallet' requires a string value, while '-nowallet' accepts only '1' to disable all wallets",
+                extra_args=['-nowallet=not_a_boolean'],
```

Comment:
In commit "init: fix fatal error on '-wallet' negated option value" (0df52e24363f729210887ca20a47824ba8073617)

I think this should be extended to test `-nowallet=0`. Maybe consider making this a loop to test both `-nowallet=0` `-nowallet=not_a_boolean`.

Reason: the simplest and most direct way to trigger the uncaught exception is to write `-nowallet=0`. The `-nowallet=not_a_boolean` case is a more complicated and indirect way to trigger it relies on `IntepretBool` interpreting the string `not_a_boolean` as `false`.

Reply:
absolutely. Pushed.